Cargo.toml

```diff
@@ -17,12 +17,12 @@ edition = "2018"
 all-features = true
 
 [dependencies]
-async-compression = { version = "0.3.1", features = ["brotli", "deflate", "gzip", "stream"], optional = true }
-bytes = "0.5"
+async-compression = { git = "https://github.com/aknuds1/async-compression", rev = "3b987d825e9ff2a50c05b9f51345170d621df23f", features = ["brotli", "deflate", "gzip", "stream", "tokio-1"], optional = true }
```

Comment:
This can now be upgraded to async-compression 0.3.7

Reply:
@paolobarbolini Looks as if that version still depends on bytes 0.5 if you use the stream feature?

Reply:
I tried building against it and couldn't make it work.

Reply:
See https://github.com/seanmonstar/reqwest/pull/1076#discussion_r548536489

What you can do is something like [this](https://github.com/seanmonstar/reqwest/pull/1076/files#diff-668dbc2595c2151fbc2dde0df240f68963eb8e30582b5cbc71cf80da947d2800), where you wrap the encoder into `tokio_util::io::StreamReader` and `tokio_util::codec::FramedRead`

Reply:
@paolobarbolini OK, do you have any tips for the remaining failures?

Reply:
[`alive_timer`](https://github.com/seanmonstar/warp/blob/5d068ae90f65cf66784c62daba8a27eaa506fa26/src/filters/sse.rs#L487) needs to be pinned, just like `event_stream` is (add `#[pin]` on top of it).

`Sleep::reset` doesn't take `self` anymore ([before](https://docs.rs/tokio/0.3.6/tokio/time/struct.Sleep.html#method.reset) - [after](https://docs.rs/tokio/1.0.0/tokio/time/struct.Sleep.html#method.reset)), so the two calls to `reset` need to be changed like so:
`first_argument.reset(second_argument)` -> `Sleep::reset(first_argument, second_argument)`

Reply:
You can still call it like a method, you just might need an `timer.as_mut().reset(arg)` in there.

Reply:
Thanks, I fixed remaining build failures. I can look at async-compression next.